tools/aie2xclbin/XCLBinGen.cpp

```diff
@@ -441,7 +441,7 @@ static LogicalResult generateXCLBin(MLIRContext *context, ModuleOp moduleOp,
         openOutputFile(aiePartitionJsonFile, &errorMessage);
     if (!aiePartitionJsonOut)
       return moduleOp.emitOpError(errorMessage);
-
+    uint64_t time = std::chrono::high_resolution_clock::now().time_since_epoch().count();
```

Comment:
Would hashing the design be more reasonable?

Reply:
In aiecc.py we use a [random int](https://github.com/Xilinx/mlir-aie/blob/main/python/compiler/aiecc/main.py#L199) with a small range, which is not super collision resistant.  Maybe hash the time plus the design?  So that parallel builds (same time) or rebuilds of the same design (e.g. after a fix) don't collide?